api-module-library/microsoft-teams/test/graph-app.test.js

```diff
@@ -7,8 +7,8 @@ describe(`${config.label} API Tests`, () => {
     const apiParams = {
         client_id: process.env.TEAMS_CLIENT_ID,
         client_secret: process.env.TEAMS_CLIENT_SECRET,
-        team_id: process.env.TEAMS_ID,
-        tenant_id: process.env.TENANT_ID,
+        team_id: process.env.TEAMS_TEAM_ID,
+        tenant_id: process.env.TEAMS_TENANT_ID,
```

Comment:
Is there any difference between a general tenant id and a teams tenant id? Asking mostly out of curiosity. Naming it with "teams" is fine given the context, and it's a test so 🤷🏼‍♂️ but did have me thinking/wondering

Reply:
I made this change to the env variable names a while ago but hadn't updated the test. It may have been based on a misunderstanding of the naming convention, that there should always be a "TEAMS" prepending the value. Probably this started with the redirect_uri